parsl/monitoring/db_manager.py

```diff
@@ -719,11 +722,12 @@ def close(self) -> None:
 
 
 @wrap_with_logs(target="database_manager")
-def dbm_starter(exception_q: "queue.Queue[Tuple[str, str]]",
-                priority_msgs: "queue.Queue[TaggedMonitoringMessage]",
-                node_msgs: "queue.Queue[MonitoringMessage]",
-                block_msgs: "queue.Queue[MonitoringMessage]",
-                resource_msgs: "queue.Queue[MonitoringMessage]",
+@typeguard.typechecked
+def dbm_starter(exception_q: mpq.Queue,
+                priority_msgs: mpq.Queue,
+                node_msgs: mpq.Queue,
+                block_msgs: mpq.Queue,
+                resource_msgs: mpq.Queue,
```

Comment:
Per the Python 3.12 gate for generic types, I ~just learned from this PR~ gather that `mpq.Queue[TaggedMonitoringMessage]` won't work; is the same also true for `[MonitoringMessage]` and the tuple of strings?  Seems a partial effort would be of some utility here, even if full coverage is 3+ years away.